File: src/models/tinybert/tinybert_repo/tb_main.py
import tb_config
from tb_export import export_to_torchscript

def main() -> None:
    """
    given nothing
    return nothing
    executes tinybert export pipeline
    """
    export_to_torchscript(tb_config.MODEL_NAME, tb_config.EXPORT_FILE)

if __name__ == "__main__":
    main()

File: src/models/tinybert/tinybert_repo/tb_main.cpp
#include <iostream>
#include "tb_config.hpp"
#include "tb_loader.hpp"

/*
given nothing
return integer exit status
loads tinybert model into memory
*/
int main() {
    torch::jit::script::Module module = load_traced_model(EXPORT_FILE);
    std::cout << "Model loaded successfully." << std::endl;
    return 0;
}
